docs/source/basics.rst

```diff
@@ -3,202 +3,147 @@
+    compiled_program = qc.compile(p)
+    result = qc.run(compiled_program)
+    print(result)
 
-    print("A composition of two programs:")
-    print(Program(X(0)) + Program(Y(0)))
+Congratulations! You just ran your program on the QVM. The returned value should be:
 
 .. parsed-literal::
 
-    Multiple inst arguments with final measurement:
-    X 0
-    Y 1
-    Z 0
-    MEASURE 0 [1]
-
-    Chained inst with explicit MEASURE instruction:
-    X 0
-    Y 1
-    MEASURE 0 [1]
-    MEASURE 1 [2]
-
-    A mix of chained inst and measures:
-    X 0
-    MEASURE 0 [1]
-    Y 1
-    X 0
-    MEASURE 0 [0]
-
-    A composition of two programs:
-    X 0
-    Y 0
-
-PyQuil can also produce a Program object by interpreting raw Quil text, as in
-the following example:
-
-.. code:: python
+    [[1]]
 
-    print(Program("X 0\nH 1\nCNOT 0 1"))
+For more information on what the above result means, and on executing quantum programs on the QVM in
+general, see :ref:`qvm`. The remainder of this section of the docs will be dedicated to constructing
+programs in detail, an essential part of becoming fluent in quantum programming.
 
-.. parsed-literal::
 
-    X 0
-    H 1
-    CNOT 0 1
+.. _standard:
 
-The ``pyquil.parser`` submodule provides a front-end to other similar parser
-functionality.
+The Standard Gate Set
+~~~~~~~~~~~~~~~~~~~~~
 
+The following gates methods come standard with Quil and ``gates.py``:
 
-Fixing a Mistaken Instruction
------------------------------
+-  Pauli gates ``I``, ``X``, ``Y``, ``Z``
 
-If an instruction was appended to a program incorrectly, one can pop it
-off.
+-  Hadamard gate: ``H``
 
-.. code:: python
+-  Phase gates: ``PHASE(theta)``, ``S``, ``T``
 
-    p = Program().inst(X(0))
-    p.inst(Y(1))
-    print("Oops! We have added Y 1 by accident:")
-    print(p)
+-  Controlled phase gates: ``CZ``, ``CPHASE00(alpha)``,
+   ``CPHASE01(alpha)``, ``CPHASE10(alpha)``, ``CPHASE(alpha)``
 
-    print("We can fix by popping:")
-    p.pop()
-    print(p)
+-  Cartesian rotation gates: ``RX(theta)``, ``RY(theta)``, ``RZ(theta)``
 
-    print("And then add it back:")
-    p += Program(Y(1))
-    print(p)
+-  Controlled :math:`X` gates: ``CNOT``, ``CCNOT``
 
-.. parsed-literal::
+-  Swap gates: ``SWAP``, ``CSWAP``, ``ISWAP``, ``PSWAP(alpha)``
 
-    Oops! We have added Y 1 by accident:
-    X 0
-    Y 1
+The parameterized gates take a real or complex floating point
+number as an argument.
 
-    We can fix by popping:
-    X 0
 
-    And then add it back:
-    X 0
-    Y 1
+Declaring Memory
+~~~~~~~~~~~~~~~~
+*Coming soon*
 
-The Standard Gate Set
----------------------
 
-The following gates methods come standard with Quil and ``gates.py``:
+Measurement
+~~~~~~~~~~~
 
--  Pauli gates ``I``, ``X``, ``Y``, ``Z``
+*Coming soon*
 
--  Hadamard gate: ``H``
 
--  Phase gates: ``PHASE(``\ :math:`\theta`\ ``)``, ``S``, ``T``
+Specifying the number of trials
+-------------------------------
 
```

Comment:
same as other comment (Is this in scope for this PR?)

I'd imagine a lot of users would like to know how to specify a number of trials